### ISSUE TITLE
Failure in `@semantic-release/npm` since v2.2.4 due to no npm in final image

### ISSUE THREAD

**Issue by ttshivers** [ahmadnassri/action-semantic-release#184]
It appears npm is no longer installed in the final image since https://github.com/ahmadnassri/action-semantic-release/commit/4937a6c324700643466610288d541c33d762a9fd This breaks `@semantic-release/npm` as it requires npm. Should npm be added back or should I adjust my workflow?

Example:
```
[9:18:12 PM] [semantic-release] › ℹ  Start step "prepare" of plugin "@semantic-release/npm"
[9:18:12 PM] [semantic-release] [@semantic-release/npm] › ℹ  Write version 1.0.21 to package.json in /github/workspace
[9:18:12 PM] [semantic-release] › ✘  Failed step "prepare" of plugin "@semantic-release/npm"
[9:18:12 PM] [semantic-release] › ✘  An error occurred while running semantic-release: Error: Command failed with ENOENT: npm version 1.0.21 --userconfig /tmp/a3b76ce75d5acead020c41967f463ea8/.npmrc --no-git-tag-version --allow-same-version
Error: Command failed with ENOENT: npm version 1.0.21 --userconfig /tmp/a3b76ce75d5acead020c41967f463ea8/.npmrc --no-git-tag-version --allow-same-version
spawn npm ENOENT
spawn npm ENOENT
    at ChildProcess._handle.onexit (node:internal/child_process:283:19)
    at onErrorNT (node:internal/child_process:476:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'spawn npm',
  path: 'npm',
  spawnargs: [
    'version',
    '1.0.21',
    '--userconfig',
    '/tmp/a3b76ce75d5acead020c41967f463ea8/.npmrc',
    '--no-git-tag-version',
    '--allow-same-version'
  ],
  originalMessage: 'spawn npm ENOENT',
  shortMessage: 'Command failed with ENOENT: npm version 1.0.21 --userconfig /tmp/a3b76ce75d5acead020c41967f463ea8/.npmrc --no-git-tag-version --allow-same-version\n' +
    'spawn npm ENOENT',
  command: 'npm version 1.0.21 --userconfig /tmp/a3b76ce75d5acead020c41967f463ea8/.npmrc --no-git-tag-version --allow-same-version',
  escapedCommand: 'npm version 1.0.21 --userconfig "/tmp/a3b76ce75d5acead020c41967f463ea8/.npmrc" --no-git-tag-version --allow-same-version',
  exitCode: undefined,
  signal: undefined,
  signalDescription: undefined,
  stdout: '',
  stderr: '',
  cwd: '/github/workspace',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false,
  pluginName: '@semantic-release/npm'
}
```

**Comment by ahmadnassri** [comment]
adding npm back today

**Comment by ahmadnassri** [comment]
fixed in [v2.2.6](https://github.com/ahmadnassri/action-semantic-release/releases/tag/v2.2.6)
